src/handlers/localstorage/accountLocal.js

```diff
@@ -205,6 +211,19 @@ export const saveLocalTransactions = (transactions, accountAddress, network) =>
     transactionsVersion
   );
 
+/**
+ * @desc save transaction signatures
+ * @param  {Object}   [transactionSignatures]
+ */
+export const saveLocalTransactionSignatures = transactionSignatures =>
+  saveLocal(TRANSACTION_SIGNATURES, transactionSignatures);
+
+/**
+ * @desc get transaction signatures
+ */
+export const getLocalTransactionSignatures = () =>
+  getLocal(TRANSACTION_SIGNATURES);
```

Comment:
also let's use `getGlobal` from `localstorage/common` and also return a default empty state value